src/pudl/metadata/resources/allocate_net_gen.py

```diff
@@ -0,0 +1,139 @@
+                "associated with each combination of generator, energy source, and "
+                "prime mover. First, the net electricity generation and fuel consumption "
+                "reported in the EIA-923 generation fuel are allocated to individual "
+                "generators. Then, these allocations are aggregated to unique generator, "
+                "prime mover, and energy source code combinations. This process does not "
+                "distinguish between primary and secondary energy_sources for generators. "
+                "Net generation is allocated equally between energy source codes, so if a "
+                "plant has multiple generators with the same prime_mover_code but different "
+                "energy source codes the generation_fuel_eia923 records will be associated "
+                "similarly between these two generators. Allocated net generation will still "
+                "be proportional to each generator's net generation or capacity."
+            ),
+            "schema": {
+                "fields": [
+                    "report_date",
+                    "plant_id_eia",
+                    "generator_id",
+                    "prime_mover_code",
+                    "energy_source_code",
+                    "energy_source_code_num",
+                    "net_generation_mwh",
+                    "fuel_consumed_mmbtu",
+                    "fuel_consumed_for_electricity_mmbtu",
+                ],
+                "primary_key": [
+                    "report_date",
+                    "plant_id_eia",
+                    "generator_id",
+                    "prime_mover_code",
+                    "energy_source_code",
+                ],
+            },
+            "field_namespace": "eia",
+            "sources": ["eia923", "eia860"],
+            "etl_group": "outputs",
+        }
+        for freq in AGG_FREQS
+    }
+    | {
+        f"generation_fuel_by_generator_{freq}_eia923": {
+            "description": (
+                f"{freq.title()} estimated net generation by generator. Based on net "
+                "generation reported in the EIA-923 generation and generation_fuel tables."
+                "The net electricity generation and fuel consumption reported in the EIA-923 "
+                "generation fuel are allocated to individual generators. This process does not "
+                "distinguish between primary and secondary energy_sources for generators. Net "
+                "generation is allocated equally between energy source codes, so if a "
+                "plant has multiple generators with the same prime_mover_code but different "
+                "energy source codes the generation_fuel_eia923 records will be associated "
+                "similarly between these two generators. Allocated net generation will still "
+                "be proportional to each generator's net generation or capacity."
```

Comment:
It's unclear to me from this description why the energy source code is involved or matters as far as the net generation allocation process. I think maybe we've forgotten to mention that the table reports fuel consumption as well as net generation?

Reply:
Took out the part about energy source codes. I think that doesn't apply here?